index.js

```diff
@@ -45,12 +45,24 @@ const submitAudit = async opts => {
   }
 };
 
+const submitDuplicate = async opts => {
+  try {
+    if (config.audit) {
+      await db('duplicates').insert(opts);
+    }
+  } catch (e) {
+    const id = opts.case_id || 'N/A (Failed to send)';
+    logError(`iCasework Case ID ${id}`, 'Duplicate', e);
+    throw new Error('Duplicate Table Error');
+  }
+};
+
 const resolver = Consumer.create({
   queueUrl: config.aws.sqs,
   handleMessage: async message => {
     const getCase = new GetCase(JSON.parse(message.Body));
     const submitCase = new SubmitCase(JSON.parse(message.Body));
-    const externalId = submitCase.get('ExternalId');
+    const externalId = getCase.get('ExternalId');
```

Comment:
I wouldn't change this as if there is any error and a case hasn't been submitted yet we won't be logging the ExternalId